test/living-dom/node-layouting.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
As noted in the contributing.md, please do not add tests in this format. Write web-platform-tests instead.